composer.json

```diff
@@ -185,7 +185,7 @@
         "doctrine/dbal": "3.3.0",
         "nikic/php-parser": "4.7.0",
         "phpunit/phpunit": "<8.0",
-        "roave/better-reflection": "<4.12.2 || >=6.0",
+        "roave/better-reflection": "<4.12.2 || >=7.0",
```

Comment:
I think we can remove this conflict, can‘t we? It does not have any effect.

Reply:
It comes from the test-case/composer.json and it has an effect there. We would have to make monorepotools aware of that in order to be able to remove this here I think.

Reply:
But I’m not sure if this would be worth the effort as this dependency was removed in Contao 5 anyway.

Reply:
Definitely not worth the effort.